__tests__/app.test.js

```diff
@@ -239,10 +243,71 @@ test('400 status response with a non existent article id', () => {
       expect(body.msg).toEqual('Not found');
   }) 
 })
+
+describe('PATCH /api/articles/:article_id', () => { 
+  test('should return a status :200 and return a body with an updated increased number of votes', () => {
+    const newVote = 10;
+    const articleVoted = {inc_votes : newVote}         
+    return request(app)
+    .patch(`/api/articles/1`)
+    .send(articleVoted)
+    .expect(200)    
+    .then(({ body }) => {    
+      for (let i = 0; i < body.length; i++) {
```

Comment:
The `body` is not an array, so can't be iterated over like you're trying to do here
This means that the expect statements in the loop aren't being reached and aren't being carried out
If the expect statements aren't reached, the test can't fail and behaves as if it's passing 🧪 
Console.log the `body` before the loop after line 255 and check what you're working with - it won't be an array, it won't have a .length property